### ISSUE TITLE
Accessibility Issue: Nested button elements

### ISSUE THREAD

**Issue by se34k** [kanriapp/kanri#926]
**Describe the bug**
Tooltip and Dropdown components have a `trigger` slot which gets wrapped into a `<button>` element. Since the trigger is a `<button>` itself most of the time, this means the rendered HTML consists of one button nested inside the other. This is an issue because (1) it is [invalid HTML](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/button#technical_summary) (since the browser deals with it just fine, this is not a major issue, but good to fix nevertheless), and (2) it degrades keyboard accessibility, since now the actual button requires two keystrokes to focus instead of just one, and the top-level button does not have any event listeners set, so users of screen readers might think they have executed an action, but have not. This is further compounded by the fact that these buttons do not have accessible names set via [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label) or `title` and their content is only an icon, so users with screen readers will only know that two buttons exist, but not what these do.

**To Reproduce**
1. Open Kanri
2. Press <kbd>Tab</kbd> to tab through interactive elements
3. Inner buttons require two tabs to get focused, <kbd>Enter</kbd> on top-level buttons does not do anything

**Expected behavior**
The actual button should get focused with just one tab press, have an accessible name and execute its associated action on pressing <kbd>Enter</kbd> or <kbd>Space</kbd>

**Screenshots**
This screenshot shows the nested DOM button structure:
<img width="964" height="723" alt="Image" src="https://github.com/user-attachments/assets/8e4754cb-f333-486b-9843-37558df1572e" />

**Desktop (please complete the following information):**
 - OS: Windows 11 (but should be the same regardless of platform)
 - App version 0.8.2

**Proposed fixes**
1. Eliminate nested buttons by either changing `DropdownMenuTrigger` and `TooltipTrigger` (I don't know where these are implemented and whether they are a third-party dependency, so this might not be possible) **or** replace explicit `<button>` with purely decorative element and move event listeners into `XTrigger` component and have the `Dropdown` and `Tooltip` component emit some event on `click`.
2. Add `aria-label` attribute to `<button>` elements to make their function accessible to screen readers.
